plugins/pencilblue/controllers/actions/admin/content/articles/edit_article.js

```diff
@@ -55,7 +55,9 @@ module.exports = function(pb) {
                 }
 
                 //TODO should we keep track of contributors (users who edit)?
```

Comment:
If you wanted to put this in I don't think it would be hard.  Just create a property ```contributors``` and make it a hash where each key is the user's ID.